.ci/daily/Jenkinsfile

```diff
@@ -82,7 +82,28 @@ pipeline {
               postSuccess: {
                 def projectVersion = cambpmUtils.getMavenProjectVersion()
                 projectVersion = projectVersion.substring(0, projectVersion.lastIndexOf('.')) + "-SNAPSHOT"
-                cambpmPublishJavadocs("\"target/site/apidocs/\"", projectVersion)
+                cambpmPublishDocs("\"target/site/apidocs/\"", projectVersion, "javadoc")
+              }
+            ])
+          }
+        }
+        stage('stage-restapi-docs') {
+          /*when {
+            branch cambpmDefaultBranch();
+          }*/
+          steps {
+            cambpmConditionalRetry([
+              agentLabel : 'centos-stable',
+              runSteps   : {
+                cambpmRunMaven('./engine-rest',
+                  "package -Dmaven.test.skip -DskipTests -pl '!engine-rest,!assembly'",
```

Comment:
Do we need `-Dmaven.test.skip` and `-DskipTests`? `maven.test.skip` should be enough. According to the [docs](https://maven.apache.org/surefire/maven-surefire-plugin/examples/skipping-tests.html), this skips compiling and running the tests and is respected by surefire.